setup.py

```diff
@@ -45,8 +45,8 @@
     "pytest-cache>=1.0,<2.0",
     "pytest-cov>=3.0.0,<4.0",
     "pytest>=6.2.5,<7.0.0",
-    "reana-commons[kubernetes]>=0.8.0a19,<0.9.0",
-    "reana-db>=0.8.0a19,<0.9.0",
+    "reana-commons[kubernetes]>=0.8.0,<0.10.0",
```

Comment:
Note: keeping lower boundary for `r-commons` and `r-db` at 0.8.0 so that we can start releasing 0.9.0a1 shared packages with fully green CI status and so that we avoid circular dependencies.

`pytest-reana` is to go first, followed by `r-commons` and `r-db`.

The lower boundary is to be bumped to `0.9.0a1` once we have `r-db` and `r-commons` released.